### ISSUE TITLE
Invalid QueryFetchAndLock.ProcessVariables type

### ISSUE THREAD

**Issue by meeshic** [citilinkru/camunda-client-go#23]
When calling QueryFetchAndLock (`POST /external-task/fetchAndLock`) with the QueryFetchAndLockTopic.ProcessVariables set, the camuda api returns the err `Object values cannot be used to query`. It seems that the ProcessVariables for the QueryFetchAndLockTopic should be a map[string]interface{}, where the value should be a string, number, or other camunda supported primitive, instead of map[string]Variable. (For Reference: [Camunda Fetch & Lock External Task API Doc](https://docs.camunda.org/manual/7.15/reference/rest/external-task/fetch/)). 

Current request payload:
```
{
    "workerId": "worker-x",
    "maxTasks": 1,
    "topics": [
	    {
	        "topicName": "topic-y",
	        "lockDuration": 100,
	        "businessKey": "8e539d03-...",
	        "processVariables": {
	            "resourceVersion": {
			"type": "String",
		        "value": "44",
			"valueInfo": {}
		    }
	        }
	    }
    ]
}
```

Suggested request payload:
```
{
    "workerId": "worker-x",
    "maxTasks": 1,
    "topics": [
	    {
	        "topicName": "topic-y",
	        "lockDuration": 100,
	        "businessKey": "8e539d03-...",
	        "processVariables": {
	            "resourceVersion": "44"
	        }
	    }
    ]
}
```

**Comment by bosskurt** [comment]
also the same with "QueryFetchAndLockTopic.TenantIdIn". I get an error with type *string. It works If I change it to []string.

**Comment by liderman** [comment]
Fixed in release v2.3.3
